IFC4x3/PropertyEnumerations/f/PEnum_FiberCableStructureType.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_FiberCableStructureType" Name="PEnum_FiberCableStructureType" UniqueId="40aa2247-7265-405c-9cab-544d272c2d3c">
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="BREAKOUT_3kYDd5QUb9fgirHkehISg" />
+		<DocPropertyConstant xsi:nil="true" href="LOOSETUBE_2bXtb4FxT2iwpMI2GTvy40" />
+		<DocPropertyConstant xsi:nil="true" href="PATCHCORD_3_59xQxz7XfnxUmc2wTAD" />
+		<DocPropertyConstant xsi:nil="true" href="PIGTAIL_05Om3rtL7QfP4iIflOyA" />
+		<DocPropertyConstant xsi:nil="true" href="TIGTHBUFFERED_1wNPLdsufCphI7kw2eY4xn" />
```

Comment:
Is this TIGHTBUFFERED ??

Reply:
Yes Miswritting. Commit : 305806220fb93c9010687f137214b3f48df310cb